package.json

```diff
@@ -28,8 +28,8 @@
         "test": "./node_modules/.bin/mocha "
     },
     "dependencies": {
-        "axios": "0.19.2",
-        "bcryptjs": "^2.4.3",
+        "axios": "^0.21.1",
+        "bcryptjs": "^5.0.0",
```

Comment:
There is no `5.x` version of `bcryptjs` - the latest is `2.4.3` which is the version we had listed already.